extensions/jitsi/common/jitsi.mjs

```diff
@@ -81,41 +94,43 @@ class JitsiHandler {
 
   /**
    * Create local track. This method would drop all current local tracks.
-   * @param isWebcam Whether the video stream should be the webcam or desktop sharing.
    */
-  async createLocalTracks(isWebcam = true) {
-    if (!isWebcam && !JitsiMeetJS.isDesktopSharingEnabled()) {
+  async createLocalTracks() {
+    if (!this.isWebcam && !JitsiMeetJS.isDesktopSharingEnabled()) {
       alert('Screen sharing is not enabled.');
     }
 
-    this.isWebcam = isWebcam;
-
     // Drop the current local tracks.
     try {
       await Promise.all(this.localTracks.map(track => track.dispose()));
+      console.log(this.room.getLocalTracks());
```

Comment:
If this is useful in actual production environment, please add log message, i.e. `console.log("...", this.room...);`
If not, please remove.